.gitignore

```diff
@@ -6,7 +6,7 @@ target/
 .attach*
 .bloop/
 .metals/
-project/metals.sbt
+**/metals.sbt
```

Comment:
uhm not sure about this change, is this automatic generated? 😅

Reply:
yep, metals generated three identical `metals.sbt`, in `project/`, `project/project/` and `project/project/project`... no idea why 🤷‍♂️ 

would you prefer `project/**/metals.sbt` maybe?

Reply:
yeah I would prefer leaving the gitignore as it is :)